packages/polaris-viz/src/components/TrendIndicator/constants.ts

```diff
@@ -4,7 +4,7 @@ export const NO_VALUE_ICON_HEIGHT = 2;
 export const TEXT_ICON_SPACING = 4;
 export const ICON_SIZE = 5.5;
 
-export const FONT_SIZE = 12;
+export const FONT_SIZE = 11;
 export const FONT_WEIGHT = 600;
```

Comment:
Can you also update `FONT_WEIGHT` and use it here https://github.com/Shopify/polaris-viz/pull/1583/files#diff-326f5604dbf454c05cf1ebf2db217bfa99e2a82d021050e2678ac2250e9615aeL70 so that the width calculation stays correct?